woocommerce/woocommerce-ios
# [IAP] Remove iap plan purchase M1 code

Closes: #10496


## Description


This PR cleans up the M1 IAP upgrades, and removes the M2 feature flag. Apologies that it's so long, it's mostly deleting the unused code though.

I've removed the strings and assets related to M1.

## Testing instructions


Using a US-based Woo Express store in a free trial
Run the app
Tap `Upgrade Now`
Explore the UI... observe that all the plan details show up correctly, and that when you start a purchase it has the correct name in Apple's UI

## Screenshots



https://github.com/woocommerce/woocommerce-ios/assets/2472348/1b2c1243-5ed5-4331-9514-0256f645a9c8


---
